### PR TITLE
 job: Expose UUID as UUID 

### PR DESCRIPTION
**Pull request by bergzand**

### CODE REVIEW

File: murdock/job.py [hunk 1]
```diff
@@ -79,6 +79,14 @@ def remove_dir(work_dir):
         except FileNotFoundError:
             LOGGER.debug(f"Directory '{work_dir}' doesn't exist, cannot remove")
 
+    @property
+    def uid(self) -> str:
+        return self._uuid.hex
```
Comment (aabadie) [inline murdock/job.py:84]:
```suggestion
        return self.uuid.hex
```

And line 88 should be covered by the tests.

Comment (bergzand) [reply murdock/job.py]: Fixed and added a test case

**Review by aabadie — APPROVED** [review]

**Comment by aabadie** [comment]
Please rebase :)

**Comment by bergzand** [comment]
Rebased!

**Pull request merged**